### DISCUSSION TITLE
Update 1.0.4

### DISCUSSION THREAD

**Discussion by dfop02** [dfop02/html4docx#3] [Announcements]
**Updates**
- Create Changelog HISTORY.
- Update README.
- Add Github Action Workflow to publish in pypi.
- Change default VERSION tag, removing the "v" from new releases.

**New Features**
- Support to internal links (Anchor) | [Dfop02](https://github.com/dfop02)

**Comment by dee-corr** [comment]
Thank you @dfop02 !! I've been struggling with html2docx defects. I'm very grateful

**Reply by dfop02** [reply]
Thank you very much @dee-corr !! I'm glad that my effort was useful for someone else
